examples/subscriptions/server.js

```diff
@@ -0,0 +1,52 @@
+const express = require('express')
+const { PubSub } = require('graphql-subscriptions')
+
+const pubsub = new PubSub()
+
+const { VoyagerServer, gql } = require('@aerogear/voyager-server')
+
+const TOPIC = 'some_topic_name'
+
+// This is our Schema Definition Language (SDL)
+const typeDefs = gql`
+  type Query {
+    hello(name: String): String
+  }
+
+  type Subscription {
+    hello: String
+  }
+`
+
+// Resolver functions. This is our business logic
+const resolvers = {
+  Query: {
+    hello: (obj, args, context, info) => {
+      const message = args.name ? `Hello ${args.name}` : 'Hello world'
+      pubsub.publish(TOPIC, { hello: message })
```

Comment:
@StephenCoady Do you think we can so some layer on top of that as separate ticket or is not worth the hassle now? Any simplifications on top of plain package that we can do as part of this epic?

Reply:
I'm not sure what you mean? Do you mean like we had in the showcase? Not sure we can simplify this for the example if thats what you mean

Reply:
Initial idea of the trello epic (if we look into it was to create subscription package that will be helping users to grasp the complexity of the thing. 
I'm curious if we actually can do more than this package now. (possibly not much apart from docs)